qa_priority_list/lambda_function.py

```diff
@@ -0,0 +1,39 @@
+import os
+from gc import gc
+from models import Projects, Tickets
+FILE_KEY = os.environ['FILE_KEY']
+SHARE_ACCOUNTS = os.environ['SHARE_ACCOUNTS']
+
+def generate_file(data, filename):
+    sh = gc.create(filename)
+    columns = ['ticket_id', 'link']
+    sh.insert_row(columns)
+    for row in data:
+        sh.insert_row(row)
+
+def share_file(sh):
+    emails = SHARE_ACCOUNTS.split(',')
+    for email in emails:
+        sh.share(email, perm_type='user', role='writer')
+
+def priority_list_by_project(project):
+
+    tickets = Tickets().get_tickets_by_project_id(project[0])
+
+    #############
+    #### TODO: implement logic to create prority list
```

Comment:
previously on 2.0 this is the sql:
```
SELECT sub.customer_email_address as Customer_email, sub.gig_description as Campaign, CONCAT('http://gigwalk.com/c/project/', sub.gig_bundle_id) as BUNDLE, 
        Date(From_unixtime(Least(sub.gig_due_by - 172800, sub.payment_create_timestamp+ 1209600))) AS QA_deadline_date, 
        Date(From_unixtime(sub.gig_due_by)) AS Bundle_due_date, 
        Date(From_unixtime(sub.payment_create_timestamp)) AS Gig_submitted_date, SUBSTRING_INDEX(sub.Location_Address, ',', 1) AS Location, 
        SUBSTRING_INDEX(sub.Location_Address, ',', -1) AS Address, sub.City_state AS City_state, CONCAT('http://gigwalk.com/c/gig/',
            sub.gig_assignment) AS GIG, sub.assignment_claimed_agent_id AS Agent_ID, agent_structured_data FROM   
        (SELECT c.customer_email_address, g.gig_description, g.gig_bundle_id, g.gig_due_by, p.payment_create_timestamp, a
            .assignment_claimed_agent_id, SUBSTRING_INDEX(l.location_formatted, ',', 2) AS Location_Address, SUBSTRING_
            INDEX(SUBSTRING_INDEX(l.location_formatted, ',', 4), ',',-2) as City_state, Concat_ws('/', a.assignment_gig_id, a
                .assignment_id) AS gig_assignment, ag.agent_structured_data FROM   v2_gigs g, v2_payments p, v2_assignm
            ents a, customer_accounts c, v2_locations l, agent_accounts ag  WHERE  g.gig_bundle_id IN ({}) AND p.payment_st
            ate_flag = 'auto_reviewed'AND g.gig_id = p.gig_id AND c.customer_id = g.gig_customer_id AND g.gig_id = a.assignmen
            t_gig_id AND l.location_gig_id = g.gig_id and ag.agent_id = a.assignment_claimed_agent_id) AS sub ORDER  
        BY QA_deadline_date, Gig_submitted_date LIMIT 2200\
```
so the logic is to prioritize based on the payment date and the gig submitted date and only take up to 2200, which should be an env. variable probably